mos_tests/nova/nova_test.py

```diff
@@ -601,3 +606,80 @@ def test_boot_instance_from_volume_bigger_than_flavor_size(self):
         # Check that instance is reachable
         ping = common_functions.ping_command(self.floating_ip.ip)
         self.assertTrue(ping, "Instance after creation is not reachable")
+
+
+class TestNovaDeferredDelete(TestBase):
+    """Nova Deferred Delete test cases"""
+
+    @classmethod
+    @pytest.fixture(scope='class', autouse=True)
```

Comment:
By default, tests will perform revert after each test. So you security group will be available only for first test.